docs/readmes/proposals/p005_call_tracing.md

```diff
@@ -0,0 +1,283 @@
+---
+id: p005_call_tracing
+title: Call Tracing for Troubleshooting
+hide_title: true
+---
+
+# Call Tracing for Troubleshooting
```

Comment:
minor nit: I guess since this includes subscriber and protocol, we can call it just Tracing support. I thought call tracing was subscriber specific

Reply:
Sure, I'm good with that!